pkgs/applications/science/electronics/csxcad/default.nix

```
@@ -0,0 +1,36 @@
+{ fetchFromGitHub, stdenv, cmake
+, fparser, tinyxml, hdf5, cgal_5, vtk
+}:
```

This is the style which the community seems to be going toward
```suggestion
{ stdenv
, fetchFromGitHub
, cmake
, fparser
, tinyxml
, hdf5
, cgal_5
, vtk
}:
```

makes diffs a lot easier to read